### PR TITLE
Interactions docs & Data update

### PR DESCRIPTION
**Pull request by DaniMarinov**

### CODE REVIEW

File: en/appbuilder/change-log.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+* Button Group
+* Text Area
+* Hyperlink
+
+### Interactions
+* Open/Close - The "Open/Close" interaction lets users create interactions using overlay components, such as Dialog.
+* Show/Hide - "Show/Hide" is an interaction that can be applied to a component to show and hide another component visibility in the same view.
+
+### Using Data in the App Builder
+* Rest API -  enables users to upload a JSON file or provide a URL with the Data Source. The added data source can be used also in the repeaters to connect components to actual data source fields.
```
Comment (kdinev) [inline en/appbuilder/change-log.md:32]: REST API means you can access a REST API endpoint. Uploading a JSON file is not a REST API. You should reword it to `Data Sources - ...`

File: en/appbuilder/change-log.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+### Code Generation
+* General improvements and bug fixes
+
+## Public Release
+
+### General
+* Option to set Min and Max Width and Height for Layouts
```
Comment (kdinev) [inline en/appbuilder/change-log.md:40]: don't capitalize.

File: en/appbuilder/change-log.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+* More components and layouts size units - The newly added units are auto min, auto max, px, rem, vh, vw, %.
+* New Layout Position options - The newly added positioning options are Relative, Absolute, Fixed, Sticky and Overflow, which can be applied to components and layouts.
```
Comment (kdinev) [inline en/appbuilder/change-log.md:42]: Don't capitalize, highlight them with bold or italic.

File: en/appbuilder/using-data-in-your-app.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 _keywords: App builder, Indigo Design, Infragistics, Data Sources, Data Binding
 ---
 
 # Data Feature overview 
-The Indigo Design App Builder Data features lets users add, edit and use external data source in the application they are developing. By default, the Indigo Design App Builder comes with a mock Northwind data source which users can use for reference. Apart from it, the data source types that can be added are JSON (Rest API), Excel or CSV files. All added data sources are placed in the Data Toolbox and users can expand/collapse each data source in order to see the included tables and selected fields.
+The Indigo Design App Builder Data features lets users add, edit and use external data source in the application they are developing. By default, the Indigo Design App Builder comes with a mock Northwind data source which users can use for reference. Currently, a Rest API data source types can be added, enabling users to upload a JSON file or provide a URL with the Data Source. All added data sources are placed in the Data Toolbox and users can expand/collapse each data source in order to see the included tables and selected fields.
```
Comment (kdinev) [inline en/appbuilder/using-data-in-your-app.md:8]: Same comment as in the Changelog section.

**Review by kdinev — CHANGES_REQUESTED** [review]

File: en/appbuilder/using-data-in-your-app.md (hunk 4 above)
Comment (DaniMarinov) [reply en/appbuilder/using-data-in-your-app.md:8]: I will leave only Northwind, Rest API, JSON capitalized since they are names.

**Review by radomirchev — APPROVED** [review]

File: en/appbuilder/using-data-in-your-app.md (hunk 4 above)
Comment (kdinev) [reply en/appbuilder/using-data-in-your-app.md]: @DaniMarinov I meant that "uploading a JSON file" is not a REST API data source.

File: en/appbuilder/change-log.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+* Button group
+* Text area
+* Hyperlink
+
+### Interactions
+* Open/Close - "Open/Close" interaction enables users create interactions using overlay components, such as <b>a dialog component</b>.
+* Show/Hide - "Show/Hide" is an interaction that can be applied to a component to show and hide another component visibility in the same view.
+
+### Using Data in the App Builder
+* Data sources -  enables users to use <b>Rest API</b> as a data source, so they can upload a JSON file or provide a URL with the data source. The added data source can be used also in the repeaters to connect components to actual data source fields.
```
Comment (kdinev) [inline en/appbuilder/change-log.md:32]: Again, uploading a JSON file is not a REST API data source.

**Review by kdinev — CHANGES_REQUESTED** [review]

File: en/appbuilder/using-data-in-your-app.md (hunk 4 above)
Comment (DaniMarinov) [reply en/appbuilder/using-data-in-your-app.md]: It should be ok now @kdinev

**Review by kdinev — APPROVED** [review]
